src/styles/tickets/frontend.pcss

```diff
@@ -3,308 +3,339 @@
+			/* /background-image: svg-inline(caret-down); */
+			height: 10px;
+			width: 10px;
+		}
 	}
-}
 
-.tribe-block__tickets__buy {
-	border: 1px solid #e1e3e6;
-	background-color: #009fd4;
-	color: #fff;
-	float: right;
-	margin: 25px 0;
+	.tribe-block__tickets__item__content__description__content {
+		display: none;
 
-	&:hover {
-		background-color: #078ab6;;
+		@media(--viewport-medium) {
+			display: block;
+		}
 	}
-}
 
-.tribe-block__tickets__item__quantity__number {
-	align-items: center;
-	background-color: transparent;
-	border: none;
-	display: inline-flex;
-	cursor: pointer;
-	outline:none;
-	justify-content: center;
-	height: 3rem;
-	margin: 0;
-	position: relative;
-	width: 3.2rem;
-	-webkit-appearance: none;
-
-	&:hover {
-		background: none;
-	}
 
-	input[type="number"] {
-		max-width: 3rem;
-		background: transparent;
-		color: #444;
-		padding: .5rem 0;
-		border: none;
-		font-size: 1.8rem;
-		height: 3rem;
-		font-weight: bold;
+
+	.tribe-block__tickets__item__extra {
+		align-items: baseline;
+		display: flex;
+		flex: 0 1 auto;
+		justify-content: left;
+		order: 2;
 		text-align: center;
-		-moz-appearance: textfield;
 
-		-webkit-appearance: textfield;
-		-moz-appearance: textfield;
-		appearance: textfield;
 
-		&::-webkit-inner-spin-button,
-		&::-webkit-outer-spin-button {
-			-webkit-appearance: none;
-			-moz-appearance: none;
-			appearance: none;
+		@media(--viewport-medium) {
+			align-items: center;
+			flex: 0 0 20%;
+			flex-flow: column;
+			order: 3;
 		}
 	}
-}
 
-/*
-	Attendee fields
-*/
+	.tribe-block__tickets__item__extra__price {
+		color: var(--color-icon-secondary);
+	}
 
-.tribe-block__tickets__item__attendee__fields {
-	background: #f9f9f9;
-	padding: 60px;
+	.tribe-block__tickets__item__extra__available {
+		margin-left: .5em;
+	}
 
-	.tribe-field {
-		margin-bottom: 40px;
-		clear: both;
-		&:last-child {
-			margin-bottom: 0;
+	.tribe-block__tickets__item__quantity {
+		flex: 0 0 90px;
+		order: 3;
+	}
+
+	.tribe-block__tickets__item__quantity__add,
+	.tribe-block__tickets__item__quantity__remove {
+		background-color: transparent;
+		color: var(--color-icon-secondary);
+		display: inline-block;
+
+		&:hover,
+		&:focus {
+			background-color: transparent;
+			color: var(--color-icon-primary);
 		}
 	}
 
-	label {
-		color: #545d66;
-		display: block;
-		font-weight: 500;
-		margin-bottom: .8em;
-
-		&.tribe-tickets-meta-field-header {
-			cursor: pointer;
-			display: inline-block;
-			float: left;
-			font-size: 15px;
-			margin-right: 20px;
+	.tribe-block__tickets__buy {
+		border: 1px solid #e1e3e6;
+		background-color: #009fd4;
+		color: #fff;
+		float: right;
+		margin: 25px 0;
+
+		&:hover {
+			background-color: #078ab6;
 		}
 	}
 
-	.tribe-tickets-meta-label {
+	.tribe-block__tickets__item__quantity__number {
+		align-items: center;
+		background-color: transparent;
+		border: none;
+		display: inline-flex;
+		cursor: pointer;
+		outline:none;
+		justify-content: center;
+		height: 3rem;
```

Comment:
rem

Reply:
Yeah, these are going away - ignore this file.